demo/scenario/Auctioneer1.createAuction.json

```diff
@@ -50,94 +50,6 @@
 		}]
 	}, {
 		"id": 2,
-		"name": "Mudberg",
```

Comment:
Removing two territories is a good simplification ... but on the other hand it does not look so "good" anymore in the UI. Is there a simple way that we keep all four territories but don't really "use" them in the auction. We dont even have to mention them. I guess the problem is that in order to finish the auction there needs to be some telco to bid for these licences. But using the scenario script this bid can just happen in the background without mentioning it at all?